messages/jsonschema/Pickle.json

```diff
@@ -25,6 +25,8 @@
       "required": [
         "astNodeIds",
         "id",
+        "keyword",
+        "keywordType",
```

Comment:
If we remove the `keyword` (see comment above), perhaps a better name for this field would be simply `type`? It describes the step just as much as it describes the keyword.

Example: This is a `Context` step.

Reply:
Yes. `keyword` is no longer in this change.